docs/reference/ml/df-analytics/apis/put-inference.asciidoc

```diff
@@ -0,0 +1,494 @@
+The index of the feature value in the feature array.
+
+`tree_node`.`node_index`:::
+(Integer) 
+The index of the current node.
+
+`tree_node`.`split_gain`:::
+(Optional, double) The information gain from the split.
+
+`tree_node`.`leaf_value`:::
+(Optional, double) 
+The leaf value of the of the node, if the value is a leaf (in other words, no 
+children).
+
+`ensemble`::
+(Optional, object)
+The definition for an ensemble model.
+
+`ensemble`.`feature_names`:::
+(Required, string) 
+Features expected by the ensemble, in their expected order.
+
+`ensemble`.`trained_models`:::
+(Required, object)
+An array of `trained_model` objects. Supported trained models are `tree` and 
+`ensemble`.
+
+`ensemble`.`classification_labels`:::
+(Optional, string) 
+An array of classification labels.
+
+`ensemble`.`target_type`:::
+(Required, string) 
+String indicating the model target type; `regression` or `classification.`
+
+`ensemble`.`aggregate_output`:::
+(Required, object) 
+An aggregated output object that defines how to aggregate the outputs of the 
+`trained_models`. Supported objects are `weighted_mode`, `weighted_sum`, and 
+`logistic_regression`.
+
+See <<ml-put-inference-model-example>> for more details.
+
+
+[[ml-put-inference-aggregated-output]]
+===== Aggregated output types
+
+`logistic_regression`::
+(Optional, object) 
+This `aggregated_output` type works with binary classification (classification 
+for values [0, 1]). It multiplies the outputs (in the case of the `ensemble` 
+model, the inference model values) by the supplied `weights`. The resulting 
+vector is summed and passed to a 
+https://en.wikipedia.org/wiki/Sigmoid_function[`sigmoid` function]. The result 
+of the `sigmoid` function is considered the probability of class 1 (`P_1`), 
+consequently, the probability of class 0 is `1 - P_1`. The class with the 
+highest probability (either 0 or 1) is then returned. For more information about 
+logistic regression, see 
+https://en.wikipedia.org/wiki/Logistic_regression[this wiki article].
+
+`logistic_regression`.`weights`:::
+(Required, double) 
+The weights to multiply by the input values (the inference values of the trained 
+models).
+
+`weighted_sum`::
+(Optional, object) 
+This `aggregated_output` type works with regression. The weighted sum of the 
+input values.
+
+`weighted_sum`.`weights`:::
+(Required, double) 
+The weights to multiply by the input values (the inference values of the trained 
+models).
+
+`weighted_mode`::
+(Optional, object) 
+This `aggregated_output` type works with regression or classification. It takes 
+a weighted vote of the input values. The the most common input value (taking the 
+weights into account) is returned.
+
+`weighted_mode`.`weights`:::
+(Required, double) 
+The weights to multiply by the input values (the inference values of the trained 
+models).
+
+See <<ml-put-inference-aggregated-output-example>> for more details.
+
+
+[[ml-put-inference-example]]
+==== {api-examples-title}
+
```

Comment:
I think it's a bit unusual to have a bunch of incomplete examples in an API reference page. I think once we have a fuller set of examples (e.g. in https://www.elastic.co/guide/en/machine-learning/master/dfanalytics-examples.html), this section should just contain a single example (even if it's an abbreviated one).